cores/arduino/stm32/pinmap.c

```diff
@@ -83,3 +83,11 @@ uint32_t pinmap_merge(uint32_t a, uint32_t b) {
     // error("pinmap mis-match");
     return (uint32_t)NC;
 }
+
+PinName pin_pinName(const PinMap* map) {
```

Comment:
I could have called map->pin directly but I preferred to create a new function. The code is more readable.